system/scripts/zone/other/character_adv.cs

```diff
@@ -710,7 +710,6 @@ private static void GrantDefaults(Character character, JobId jobId)
 			{
 				LearnSkill(character, SkillId.Hammer_Attack);
 				LearnSkill(character, SkillId.Hammer_Attack_TH);
-				LearnSkill(character, SkillId.Sadhu_OutofBodyCancel);
```

Comment:
Can you tell me why you removed this? It's part of the official data, which there probably is a reason for.

Reply:
This skill is not being used anymore and has no purposes. Sadhu has been remake I think.

Reply:
This is based on official data though, and removing it creates a dissonance between server and client. Unless this causes issues, it should stay this way.